### ISSUE TITLE
Wallchart not displaying additional awards

### ISSUE THREAD

**Issue by 2FCubs** [NomisNostab/Topo-Blazor#183]
The Wallchart is not displaying completed walkabout awards. The additional award report is working correctly.

[Additional_Awards_Cub_Scouts-4.pdf](https://github.com/user-attachments/files/17008506/Additional_Awards_Cub_Scouts-4.pdf)
[Wallchart_Cub_Scouts-18.pdf](https://github.com/user-attachments/files/17008507/Wallchart_Cub_Scouts-18.pdf)
 .

**Comment by NomisNostab** [comment]
Hi,
On the additional rewards report could you show me the Nights and KMs values after the kids names?
Simon.

**Comment by 2FCubs** [comment]
Hi Simon,
Apologies for the delay, here is a clean version

With thanks,
Rebecca


________________________________

**Comment by NomisNostab** [comment]
Hi Rebecca,
I think the internet pixies stole the attachment.
Simon.

**Comment by NomisNostab** [comment]
Hi Rebecca,
Also run a logbook report for the kids as well.
That will show what has been recorded in the log book for nights and KMs.
These are the values that I use on the wallchart.
Simon.

**Comment by 2FCubs** [comment]
Hi Simon,
Correct awards report attached. I printed the logbook but it only has one cub scout. I suspect the wall chart isn't showing anything because all the cubs have 0KMs and 0 nights except for Joshua? Why don't bulk achievements show up in the logbook?

Thanks
Rebecca
________________________________

**Comment by NomisNostab** [comment]
Rebecca.
The Achievement for the KMs and Nights are not connected to the log book. You would use the log book to work out when to give a badge.
What I did when we moved to terrain was to create a carry over entry in each members log book.
It was a hike for how ever many nights they have camped and KMs walked. Make sure these count towards achievement.
It's a bit of a pain, but once done you will have a base that any additional log book entries will add to.
Simon.

PS. The attachments still didn't come through, but don't worry as I think we have worked out the root cause.
